python/lsst/pex/config/configField.py

```diff
@@ -186,8 +186,8 @@ def save(self, outfile, instance):
         ----------
         outfile : file-like object
             A writeable field handle.
-        instance : `Config`
-            The `Config` instance that contains this field.
+        instance : `~lsst.pex.config.Config`
```

Comment:
I am baffled that this did not work previously.  `lsst.pex.config.configField` (this file) and `lsst.pex.config.config` (where `Config` lives) both have all of their symbols lifted into `lsst.pex.config`, and I think our docs wouldn't have any links at all if had to use the private module names rather than the packages they are linked to.

Reply:
This is related to your other comment. The ConfigurableAction stuff is not visible in the `lsst.pex.config` namespace. This means that when they inherit these methods sphinx can't work out what `Config` means. Fully qualifying the class lets it work.

Reply:
But there's nothing involving the configurable actions classes or packages here at all - it seems to be a straightforward reference to a sibling module whose symbols are lifted to package scope, like, say, `DatasetRef` docs referencing `DatasetType` without having to say `~.type.DatasetType` or `~lsst.daf.butler.DatasetType`.

Reply:
`lsst.pex.config` and `lsst.pex.config.configurableActions` are different modules. When we refer to `Config` in `lsst.pex.config.configurableActions` (because we inherited this method) it doesn't know it's referring to `lsst.pex.config.Config` and thinks it is referring to `lsst.pex.config.configurableActions.Config` which does not exist. Specifying the full path in the docstring of the base class allows the subclass to pick up the right class.

Reply:
To be clear, this is not a problem with the rendering of anything in `configField`, it's entirely down to what the other module thinks it's getting.

Reply:
Ah, ok, I hadn't realized it as being about making the base class docstring workable when inherited by a subclass.  Shame that's not cleaner, but it's certainly not the kind of problem I'd have anticipated when writing a documentation-scraper, either.  Thanks for the explanation.